c2cgeoportal/models.py

```diff
@@ -1017,7 +1017,8 @@ class Dimension(Base):
     def __init__(self, name="", value="", layer=None):
         self.name = name
         self.value = value
-        self.layer = layer
+        if layer is not None:
+            self.layer = layer
```

Comment:
If I set the layer to `None` form-alchemy will not update it with the form data :-(